core/tests/protractor_desktop/explorationHistoryTab.js

```diff
@@ -536,35 +537,93 @@ describe('Exploration history', function() {
     historyGraph.expectHistoryStatesToMatch(expectedHistoryStates);
     historyGraph.expectNumberOfLinksToMatch(2, 0, 0);
 
-    // Check that reverting works.
+    users.logout();
+  });
+
+  it('should revert to old commit', function() {
```

Comment:
`should revert to old exploration commit`

Reply:
Done!